src/components/Vuetify/mixins/breakpoint.ts

```diff
@@ -1,90 +1,62 @@
 import Vue from 'vue'
 import { VuetifyBreakpoint } from 'types'
 
+const breakpoints = {
+  xs: 600,
+  sm: 960,
+  md: 1280 - 16,
+  lg: 1920 - 16,
+  xl: Infinity
+}
+const keys = Object.keys(breakpoints) as (keyof typeof breakpoints)[]
+const values = Object.values(breakpoints)
+type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
+type Static = Omit<VuetifyBreakpoint, 'width' | 'height'>
+const breakpointExplainations: Static[] = keys.map((name, rowIndex) => {
+  const result: Record<
+  string,
+  keyof Omit<VuetifyBreakpoint, 'width' | 'height'> | boolean
+  > = { name }
+  keys.forEach((key: keyof typeof breakpoints, i) => {
```

Comment:
`key` should be inferred if `keys` has the correct type.

Reply:
Donezo. At some point it didn't used to get inferred, it does now.